arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/signals/Exceptions.java

```diff
@@ -45,6 +45,21 @@ private Exceptions() {}
   public static final short INVALID_CODE_PREFIX = 512;
   public static final short CODE_SIZE_OVERFLOW = 2048;
 
+  public static boolean isException(short exception) {
```

Comment:
we already have a method `isAny`, we don't need this duplicates method

Reply:
Also, if anything, the name should be "is pure exception" as if you have confluent exceptions your `exception` would look like e.g. `0b00101100`, here you are checking if `exception` looks like e.g. `0b00000100`